pkg/lwip/include/arch/cc.h

```diff
@@ -8,7 +8,7 @@
 
 /**
  * @defgroup    lwip_arch_cc    Compiler and processor description
```

Comment:
`pkg_lwip_arch_cc`?

Reply:
Ok, but a bit out of the scope of this PR. If you insist, I can also change that.

Reply:
It's less out of scope than the introduction of `auto_init` doc below since this PR is about group naming ;-).

Reply:
Ok, you insist ;)